cmd/utils/nodecmd/chaincmd.go

```diff
@@ -164,6 +178,17 @@ func initGenesis(ctx *cli.Context) error {
 	return nil
 }
 
+func dumpGenesis(ctx *cli.Context) error {
+	genesis := utils.MakeGenesis(ctx)
+	if genesis == nil {
+		genesis = blockchain.DefaultGenesisBlock()
+	}
+	if err := json.NewEncoder(os.Stdout).Encode(genesis); err != nil {
+		utils.Fatalf("could not encode genesis")
```

Comment:
How about using `logger.Crit` instead of `utils.Fatalf`?

Reply:
I reflected it.
Sorry for late.